go.mod

```
@@ -37,9 +37,12 @@ require (
 )
 
 replace (
-	// fixes pkg/mod/github.com/golang/glog@v1.1.0/internal/logsink/logsink.go:123:41: undeclared name: any (requires version go1.18 or later)
-	// remove when updating to go lang 1.19
-	github.com/golang/glog => github.com/golang/glog v1.0.0
+	// The x/crypto is pulled from golang.org/x/net@v0.10.0
+	// Fixes CVE-2023-48795 - golang.org/x/crypto Authentication Bypass by Capture-replay
+	golang.org/x/crypto => golang.org/x/crypto v0.17.0
 	golang.org/x/net => golang.org/x/net v0.17.0
+	// remove when upgrade to controller-runtime 0.15.xor apimachinery to 0.27.x
+	// Fixes github.com/elazarl/goproxy Denial of Service (DoS)
+	// This dependency was remove from apimachinery 0.27.0
```

Just a suggestion, I think it's a bit more succinct for whoever handles it next.

```suggestion
	
	// Fixes github.com/elazarl/goproxy Denial of Service (DoS), the dependency was removed in v0.27.0
	// Remove below when upgrading to controller-runtime 0.15.x or apimachinery to 0.27.x
```

---

cool.
applied, thx.